ActindoWorkFlow/Overview/04_WorkflowProcessElements.md

```diff
@@ -36,54 +37,57 @@ Workflow processes, as well as the workflows itself, consists of certain element
 
 - *Task event*  
-    Click the drop-down list and select the applicable task event. All available task events are displayed. This option is linked to the *Tasks* module. The drop-down list is only displayed if the *Tasks* module is installed and the user has the necessary rights.
+    Click the drop-down list and select the applicable task event. All available task events are displayed. This option is linked to the *Tasks* module. The drop-down list is only displayed if the *Tasks* module is installed and the user has the necessary rights. For detailed information see [Manage the events](../../Tasks/Operation/01_ManageEvents.md) in the *Tasks* documentation.
 
     > [Info] The *ActindoWorkflow Process Execution Failed* task is preconfigured by default. A list of all existing task events is displayed in the *Events* menu entry of the *Tasks* module.
 
+- *Long description*    
+    If desired, edit a long text to document the transition. The *Edit long description* window is displayed. You can add a long text using a simple markdown editor.
+
 - *Configuration*  
     Depending on the transition, there might be configuration options, for instance, defining an email where a receipt or document must be sent. Editing the transition configuration may require basic programming knowledge. Most transitions are preconfigured and do not include configuration options. In this case, the *Configuration* section is not displayed.
 
 - *Static inputs*  
-    All unlinked input ports are listed in this section. Static inputs can be used to configure static values for certain inputs. They can be defined by the user clicking the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button. Only valid JSON values are accepted. When used, static inputs are displayed by a small arrow in front of the input ports in the workflow diagram. This section is only displayed, if the selected transition has at least one unlinked input port.
-
-    > [Info] Not all input ports can be configured as static inputs. At least one input port must be linked to a place.
-
-When a transition is executed in a workflow process, it is called action or process action. This means that an action is the actual execution of a configured transition. As a result, one transition may be associated with multiple process actions.   
-For detailed information, see [Actions](#actions).
+    All unlinked input ports are listed in this section. You can use static inputs to configure static values for certain inputs. You can define it by clicking the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button. Only valid JSON values are accepted, for example text ("text"), numbers ("42") or JSON objects ({"dispatchNote": {"id": 42}}). When used, static inputs are displayed by a small arrow in front of the input ports in the workflow diagram. This section is only displayed, if the selected transition has at least one unlinked input port.
```

Comment:
Vorschlag: You can define a static value by clicking the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button next to the desired static input. (oder so, sonst fehlt der Bezug, finde ich)